geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneQueriesPersistenceIntegrationTest.java

```diff
@@ -48,13 +49,13 @@
 public class LuceneQueriesPersistenceIntegrationTest extends LuceneIntegrationTest {
 
   @Rule
-  public DiskDirRule diskDirRule = new DiskDirRule();
+  public TemporaryFolder tempFolderRule = new TemporaryFolder(new File("."));
```

Comment:
Yeah. That's a good idea. Turns out there is an empty TemporaryFolder constructor, so I'll make that change.